.github/workflows/ruby-qltest.yml

```diff
@@ -17,10 +20,14 @@ on:
       - "shared/**"
       - .github/workflows/ruby-qltest.yml
       - .github/actions/fetch-codeql/action.yml
+      - .github/actions/cache-query-compilation/action.yml
+      - .github/actions/reasonable-memory/action.yml
+      - csharp/actions/create-extractor-pack/action.yml
       - codeql-workspace.yml
     branches:
       - main
       - "rc/*"
+  workflow_dispatch:
```

Comment:
It's really handy to be able to test these workflows (especially given that they have branch constraints that prevent normal testing)